CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 # Change Log
 
+## 1.12.0
+
+Bug fixes:
+
+- Fixed opening initial popup in recent Chrome versions
+  ([#317](https://github.com/oasisprotocol/oasis-wallet-ext/pull/317)).
+
 ## 1.11.0
 
 Spotlight change:
 
 - Encourage decentralization by shuffling validators on every load
   ([#309](https://github.com/oasisprotocol/oasis-wallet-ext/pull/309)).
+- Warn about not storing mnemonic in a future version
```

Comment:
Why it is listed under 1.11 where it was not part of the prev release?

Reply:
oops, I checked the feature and version in a dev build instead of release